testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/ReadOnlyModeTestCase.java

```diff
@@ -120,13 +135,34 @@ public void testReadOnlyConfigurationDirectory() throws Exception {
             }
 
         } finally {
+            if (TestSuiteEnvironment.isWindows()) {
+                UserPrincipal owner = Files.getFileAttributeView(configDir, FileOwnerAttributeView.class).getOwner();
+                AclEntry entry = AclEntry.newBuilder()
+                        .setPrincipal(owner)
+                        .setType(AclEntryType.ALLOW)
+                        .setPermissions(AclEntryPermission.WRITE_DATA, AclEntryPermission.APPEND_DATA)
+                        .build();
+                AclFileAttributeView view = Files.getFileAttributeView(roConfigDir, AclFileAttributeView.class);
+                List<AclEntry> acl = view.getAcl();
+                acl.add(0, entry);
+                view.setAcl(acl);
+            } else {
+            Set<PosixFilePermission> perms = new HashSet<>();
             perms.add(PosixFilePermission.OWNER_WRITE);
             perms.add(PosixFilePermission.GROUP_WRITE);
             perms.add(PosixFilePermission.OTHERS_WRITE);
-
+            perms.add(PosixFilePermission.OWNER_READ);
+            perms.add(PosixFilePermission.OWNER_EXECUTE);
+            perms.add(PosixFilePermission.GROUP_READ);
+            perms.add(PosixFilePermission.GROUP_EXECUTE);
+            perms.add(PosixFilePermission.OTHERS_READ);
+            perms.add(PosixFilePermission.OTHERS_EXECUTE);
             Files.getFileAttributeView(roConfigDir, PosixFileAttributeView.class).setPermissions(perms);
-
+        }
             PathUtil.deleteRecursively(roConfigDir);
+            if( TestSuiteEnvironment.isWindows()) {
```

Comment:
I think the delete should be left out, as long as the permissions are reset this will get deleted by normal cleanup and shouldn't have any issues, and a recursive delete floating around on windows gives me the heebee-jeebies :)

Reply:
ok, removed